src/realm_dart_decimal128.cpp

```diff
@@ -25,13 +25,13 @@ RLM_API realm_decimal128_t realm_dart_decimal128_from_string(const char* string)
     return to_decimal128(result);
 }
 
+// This buffer is reused between calls, hence it is thread local
+thread_local char _decimal128_to_string_buffer[34];
```

Comment:
This appears to be pulled from another PR.

Reply:
Yes, sorry.. will fix.